<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="utf-8">
		<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
		<meta name="viewport" content="width=device-width, initial-scale=1">
		
		<meta name="author" content="Peter Klipfel">
		<meta name="description" content="My portal minimalistic portal to the world">
		<meta name="generator" content="Hugo 0.59.1" />
		<title>Polymerase Chain Reaction (PCR) &middot; Peter Klipfel</title>
		<link rel="shortcut icon" href="http://peterklipfel.com/images/favicon.ico">
		<link rel="stylesheet" href="http://peterklipfel.com/css/style.css">
		<link rel="stylesheet" href="http://peterklipfel.com/css/highlight.css">

		
		<link rel="stylesheet" href="http://peterklipfel.com/css/font-awesome.min.css">
		

		
		<link href="http://peterklipfel.com/index.xml" rel="alternate" type="application/rss+xml" title="Peter Klipfel" />
		

		
	</head>

    <body>
       <nav class="main-nav">
	
	
		<a href='http://peterklipfel.com/'> <span class="arrow">←</span>Home</a>
	
  
	<a href='http://peterklipfel.com/tags'>Tags</a>
	<a href='http://peterklipfel.com/about'>About</a>

	

	
	<a class="cta" href="http://peterklipfel.com/index.xml">Subscribe</a>
	
</nav>


        <section id="wrapper" class="post">
            <article>
                <header>
                    <h1>
                        Polymerase Chain Reaction (PCR)
                    </h1>
                    <h2 class="headline">
                    Nov 27, 2019 21:41
                    · 1853 words
                    · 9 minute read
                      <span class="tags">
                      
                      </span>
                    </h2>
                </header>
                
                  
                    <div id="toc">
                      <nav id="TableOfContents">
<ul>
<li><a href="#why-do-synthetic-biologists-use-pcr">Why do synthetic biologists use PCR?</a></li>
<li><a href="#how-does-it-work">How does it work?</a>
<ul>
<li><a href="#recipe">Recipe</a>
<ul>
<li>
<ul>
<li><a href="#ingredients">Ingredients</a></li>
<li><a href="#equipment">Equipment</a></li>
<li><a href="#steps">Steps</a></li>
<li><a href="#testing-your-dna">Testing your DNA</a></li>
</ul></li>
</ul></li>
<li><a href="#questions-that-are-still-unanswered">Questions that are still unanswered</a>
<ul>
<li><a href="#what-s-going-on-when-the-thermocycler-changes-temperatures">What&rsquo;s going on when the thermocycler changes temperatures?</a>
<ul>
<li><a href="#denaturation">Denaturation</a></li>
<li><a href="#annealing">Annealing</a></li>
</ul></li>
<li><a href="#how-do-i-create-primers">How do I create primers?</a></li>
<li><a href="#how-do-i-extract-dna">How do I extract DNA?</a></li>
</ul></li>
<li><a href="#pitfalls">Pitfalls</a>
<ul>
<li><a href="#too-few-cycles">Too few cycles</a></li>
<li><a href="#denaturation-is-too-long">Denaturation is too long</a></li>
<li><a href="#denaturation-is-too-short">Denaturation is too short</a></li>
<li><a href="#annealing-temperature-is-too-high">Annealing temperature is too high</a></li>
<li><a href="#annealing-is-too-short">Annealing is too short</a></li>
<li><a href="#extension-is-not-at-optimal-temperature">Extension is not at optimal temperature</a></li>
<li><a href="#extension-is-too-short">Extension is too short</a></li>
<li><a href="#dntp-concentration-was-wrong">dNTP concentration was wrong</a></li>
<li><a href="#too-many-gs-and-or-cs-in-the-target-gene">Too many Gs and/or Cs in the target gene</a></li>
<li><a href="#contaminants-in-the-ingredients">Contaminants in the ingredients</a></li>
<li><a href="#incorrect-amounts-of-primer">Incorrect amounts of primer</a></li>
<li><a href="#too-little-polymerase">Too little polymerase</a></li>
<li><a href="#primers-were-designed-incorrectly">Primers were designed incorrectly</a></li>
<li><a href="#buffer-concentration-was-too-low">Buffer concentration was too low</a></li>
<li><a href="#target-was-too-long">Target was too long</a></li>
<li><a href="#source-dna-is-bad">Source DNA is bad</a></li>
</ul></li>
</ul></li>
<li><a href="#other-sources">Other sources</a></li>
</ul>
</nav>
                    </div>
                  
                
                <section id="post-body">
                    

<h1 id="why-do-synthetic-biologists-use-pcr">Why do synthetic biologists use PCR?</h1>

<p>One of the foundations of synthetic biology is being able to put new genes into organisms that did not previously have them. Unfortunately, genes don&rsquo;t naturally come in buckets of purified DNA segments; we have to make those ourselves. And this is where PCR comes in. If you happened to find a really cool jelly fish that glowed under UV light, you sequenced that jellyfish&rsquo;s genome, and identified the gene that makes it glow, you might be able to take that gene from the jellyfish and add it to other organisms in order to make them glow as well. Let&rsquo;s assume for this explanation that we have already identified the gene that causes the jellyfish to glow, and that we have a tank full of these jellyfish.</p>

<p>Now you have all the genes you need, if you could only extract them from the jellyfish somehow. This is a separate process called DNA extraction, and I&rsquo;m going to save that for a different post, but there do exist ways to extract DNA from the cells of an organism. Once you have the DNA extracted, you still need to get a bunch of copies of the specific gene that you want to inject. If you just injected your purified DNA&hellip; well, I don&rsquo;t know what would happen, and this might be a fun experiment, but my guess is that you probably wouldn&rsquo;t get anything useful.</p>

<p>Back to the original plan of extracting the gene from the jellyfish. If we have a tube full of purified jellyfish DNA, PCR is the process that will turn the tube full of purified DNA into a tube full of a specific gene, which we can then use in combination with the DNA of the target organism to produce a new genome for the target organism with our modified gene in it.</p>

<h1 id="how-does-it-work">How does it work?</h1>

<p>I like to approach synthetic biology from a recipe perspective. That is, listing out the ingredients, and then how to put them together.</p>

<h2 id="recipe">Recipe</h2>

<h4 id="ingredients">Ingredients</h4>

<ul>
<li>Primer 1</li>
<li>Primer 2</li>
<li>Source DNA</li>
<li>Buffer

<ul>
<li>Used primarily to maintain the correct pH levels in the reaction so that the other ingredients don&rsquo;t break apart into their constituent atoms or hydrocarbons</li>
</ul></li>
<li>dNTPs (dATP, dGTP, dCTP, and dTTP)

<ul>
<li>These are the chemical acronym versions of the A, T, C, and G that make up the DNA</li>
</ul></li>
<li>Polymerase

<ul>
<li>Taq polymerase is the one that people like to talk about, but there are lots of different kinds of polymerases which have tradeoffs like how much DNA it will copy at lower temperatures (when you don&rsquo;t want it to), price, and error rates.</li>
</ul></li>
</ul>

<h4 id="equipment">Equipment</h4>

<ul>
<li>Thermocycler</li>
<li>PCR tubes</li>
<li>Pipette</li>
</ul>

<h4 id="steps">Steps</h4>

<ol>
<li>Use the pipette to add together all the ingredients in a PCR tube</li>
<li>Put the PCR tube with the mixed ingredients into the thermocycler</li>
<li>Run the thermocycler (The cycle that follows uses example temperatures for Taq polymerase. Different polymerases and primers will perform optimally at different temperatures)

<ol>
<li>Heat to high temperature 96° C for 4 minutes</li>
<li>Heat to high temperature 96° C for 30 seconds</li>
<li>Cool to 59° C for 30 seconds</li>
<li>Heat to 72° C for 3 minutes</li>
<li>Repeat starting from step 2 30 times</li>
</ol></li>
<li>Remove the PCR tubes from the thermocycler</li>
</ol>

<p>Notice that some of the steps take longer than others. These can all be tweaked to create an optimally performing process. It is also worth mentioning that primers should only be expected to one site or the other in a DNA sequence. Sometimes, both may bind, but in general, it&rsquo;s useful to assume that only one will bind. This means that in the second cycle there will be a longer tail of DNA on either side of the primers (but only for one side of the DNA on each end). As the cycle progresses, these extra bits will cleave off, and the desired gene will be more and more perfect. This is why PCR cycles many times.</p>

<h4 id="testing-your-dna">Testing your DNA</h4>

<p>The first way that you will want to test your DNA is with gel electrophoresis. I&rsquo;m not going to go into the details here, but it enables you to see the length(s) of DNA that you created in your PCR steps. If the length is what you expect, that&rsquo;s pretty good indicator that you got what you wanted.</p>

<h2 id="questions-that-are-still-unanswered">Questions that are still unanswered</h2>

<h3 id="what-s-going-on-when-the-thermocycler-changes-temperatures">What&rsquo;s going on when the thermocycler changes temperatures?</h3>

<p>The thermocycler is going through 3 phases:</p>

<ol>
<li>Denaturation</li>
<li>Annealing</li>
<li>Extension</li>
</ol>

<h4 id="denaturation">Denaturation</h4>

<p>DNA is made of two strands, and these strands can be broken apart at high temperature. The first step of PCR is to heat up the DNA such that the two strands are separated. This is why special polymerases are chosen; they must be able to withstand very high temperatures. Taq, the first such polymerase was found in an organism that lives on deep sea vents where water temperature is very close to boiling.</p>

<h4 id="annealing">Annealing</h4>

<p>Now that the two strands of DNA are separated, the primers have sites to bind to. When the DNA was stuck together, there was no room for each of the primers. The temperature of the annealing step should be lower than the extension temperature so that the polymerase doesn&rsquo;t start extending the DNA before the primers are done binding. The binding of primers will force the strands to start and stop at the right places.</p>

<h3 id="how-do-i-create-primers">How do I create primers?</h3>

<p>There are two primers that are used in PCR. The forward primer and the reverse primer. I think of these as the &ldquo;start primer&rdquo; and the &ldquo;end primer&rdquo;. There are a lot of things to keep in mind when designing these two primers like primer length, G/C content, annealing temperature, structural gotchas (hairpin and dimer sequences). Rather than explaining them all here, I think Premier Biosoft has a great <a href="http://www.premierbiosoft.com/tech_notes/PCR_Primer_Design.html">PCR primer design summary</a></p>

<p>In general, the forward primer should start with methionine which is represented by the codon ATG. And the reverse primer should end with TAG, TAA, TGA</p>

<p>There are a couple of pieces of software that help with primer design:</p>

<ul>
<li><a href="http://www.genomecompiler.com/">Genome Compiler</a> is quite popular</li>
<li>I like <a href="https://benchling.com">Benchling</a></li>
<li>I hear that <a href="https://www.snapgene.com/">Snapgene</a> is used widely in industry</li>
</ul>

<h3 id="how-do-i-extract-dna">How do I extract DNA?</h3>

<p>DNA extraction depends on the type of organism you want to extract the DNA from, and I&rsquo;m going to leave this subject for another post (or series of posts)</p>

<h2 id="pitfalls">Pitfalls</h2>

<p>I thought BioRad had a pretty good <a href="https://www.bio-rad.com/en-us/applications-technologies/pcr-troubleshooting">overview of PCR troubleshooting</a>. I&rsquo;ve summarized some of the parts of that article, and synthesized them with other findings, and my own thoughts.</p>

<h3 id="too-few-cycles">Too few cycles</h3>

<p>Each cycle runs the full amplification process through once, so if too few of these cycles are run, there will be too little generated DNA sequences. The resulting solution will have too much of the original ingredients, and too little of the final product (the gene that&rsquo;s being amplified).</p>

<h3 id="denaturation-is-too-long">Denaturation is too long</h3>

<p>If denaturation runs too long, the DNA will start to fall apart. It is difficult to copy DNA that has fallen apart.</p>

<h3 id="denaturation-is-too-short">Denaturation is too short</h3>

<p>If denaturation does not have a chance to fully break apart the DNA, the primers will have no place to bind, and the annealing step will be ineffective. This can cause poor amplification.</p>

<h3 id="annealing-temperature-is-too-high">Annealing temperature is too high</h3>

<p>Primers will fail to bind to the DNA. This will cause poor amplification, and could cause other problems as well.</p>

<h3 id="annealing-is-too-short">Annealing is too short</h3>

<p>The primers will not have enough time to bind to the DNA. This can cause a couple problems because extension will just produce copies of random pieces of the DNA.</p>

<h3 id="extension-is-not-at-optimal-temperature">Extension is not at optimal temperature</h3>

<p>Different polymerases will perform optimally at different temperatures. If the temperature is not optimal, the extension reaction may take too long, and this could lead to the problem of the extension time being too short.</p>

<h3 id="extension-is-too-short">Extension is too short</h3>

<p>The process will produce fragmented DNA because the polymerase will not have enough time to extend the segments of DNA before it is denatured again.</p>

<h3 id="dntp-concentration-was-wrong">dNTP concentration was wrong</h3>

<p>If there is too many dNTP in the solution, the buffer will run out of ions to maintain the pH level with, and the pH of the solution will become unstable.</p>

<p>If there is too little dNTP in the solution, there may be inadequate amplification because there won&rsquo;t be enough DNA building blocks to amplify the gene with.</p>

<h3 id="too-many-gs-and-or-cs-in-the-target-gene">Too many Gs and/or Cs in the target gene</h3>

<p>Gs and Cs are &ldquo;sticky&rdquo;, and if there are a lot of them, it becomes difficult to amplify them without ruining the source DNA.</p>

<h3 id="contaminants-in-the-ingredients">Contaminants in the ingredients</h3>

<p>If there are contaminants in any of the ingredients the reaction will not work right. The first thing to try is to be extra careful with pipetting to make sure that there was no user error. If you are pretty positive that no user errror has occured, then it might be time to order new ingredients. Debugging contaminated ingredients can be pretty tricky. If you have ingredients from other reactions, it might be worthwhile to try some control reactions with other known-good ingredients.</p>

<h3 id="incorrect-amounts-of-primer">Incorrect amounts of primer</h3>

<p>If there is too much primer, the primers have a higher likelihood of binding to things that they aren&rsquo;t supposed to be binding to. This can cause the final solution to have extra DNA fragments that shouldn&rsquo;t be there.</p>

<p>If there is not enough primer, then PCR will be inefficient because the start and end points of the desired duplication will not be marked. This can lead to non-specific amplification by the polymerase, or it can simply lead to an inefficient process.</p>

<p>The amount of primer must be balanced because at the beginning of the reaction, there will be a higher primer to DNA ratio, than at the end. As the DNA is duplicated further, the dNTPs are used up to create more DNA that the primers need to bind to.</p>

<h3 id="too-little-polymerase">Too little polymerase</h3>

<p>Similarly to the primer, if there is not enough polymerase, the reaction will be inefficient. And this must be balanced across the length of the reaction.</p>

<h3 id="primers-were-designed-incorrectly">Primers were designed incorrectly</h3>

<p>If the primers are wrong, there may be mutations in the genome that were not intended. The primers may bind to unintended parts of the source DNA. This conclusion should probably come toward the end of the debugging process, but may be indicated by DNA lengths being consistently wrong.</p>

<h3 id="buffer-concentration-was-too-low">Buffer concentration was too low</h3>

<p>Not enough buffer can cause the pH of the solution to be unstable and cause ineffective PCR.</p>

<h3 id="target-was-too-long">Target was too long</h3>

<p>The simplest way to get around this is to try to run the reaction longer. Otherwise, maybe try to find other ingredients that are better suited to your target, or try to optimize the target to be shorter.</p>

<h3 id="source-dna-is-bad">Source DNA is bad</h3>

<p>If the source DNA is of poor quality, then the reaction will not run correctly. Fresher DNA has higher quality.</p>

<h1 id="other-sources">Other sources</h1>

<ul>
<li>Khan academy has some amazing articles. I found the <a href="https://www.khanacademy.org/science/biology/biotech-dna-technology/dna-sequencing-pcr-electrophoresis/a/polymerase-chain-reaction-pcr">PCR post</a> to be well written</li>
<li>The Thought Emporium has an excellent video about PCR titled <a href="https://youtu.be/7kJ2o7P8D00">The complete guide to PCR</a></li>
</ul>

                </section>
            </article>

            

            
                <div id="disqus_thread"></div>
<script type="text/javascript">
    var disqus_shortname = 'peterklipfel'; 

     
    (function() {
        var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
        dsq.src = '//' + disqus_shortname + '.disqus.com/embed.js';
        (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
    })();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
</div>

            

            

            <footer id="footer">
    
        <div id="social">

	
	
    <a class="symbol" href="https://www.github.com/peterklipfel">
        <i class="fa fa-github-square"></i>
    </a>
    


</div>

    
    <p class="small">
    
       © Copyright 2020
       <br>
       <i class="fa fa-flask" aria-hidden="true"></i> Peter Klipfel
    
    </p>
</footer>

        </section>

        <script src="http://peterklipfel.com/js/jquery-3.3.1.min.js"></script>
<script src="http://peterklipfel.com/js/main.js"></script>
<script src="http://peterklipfel.com/js/highlight.min.js"></script>
<script>hljs.initHighlightingOnLoad();</script>







    </body>
</html>
